dependencies.tsv

```diff
@@ -99,23 +100,26 @@ gopkg.in/amz.v3	git	8c3190dff075bf5442c9eedbf8f8ed6144a099e7	2016-12-15T13:08:49
 gopkg.in/check.v1	git	4f90aeace3a26ad7021961c297b22c42160c7b25	2016-01-05T16:49:36Z
 gopkg.in/errgo.v1	git	442357a80af5c6bf9b6d51ae791a39c3421004f3	2016-12-22T12:58:16Z
 gopkg.in/goose.v2	git	4dc44b23a313a8a3e4051e7ee568a954a8cb3385	2018-02-08T12:05:40Z
+gopkg.in/httprequest.v1	git	1a21782420ea13c3c6fb1d03578f446b3248edb1	2018-03-08T16:26:44
```

Comment:
We import this in juju code base as
"github.com/juju/httprequest"

So we should look to be consistent. Perhaps we should migrate to using the gopkg.in import in juju.
But regardless, we don't want this extra dep. I think we can just drop it here actually on second thoughts.

Reply:
gopkg.in/httprequest.v1 is used in juju/utils (recent commit) and also in charmstore.v5
So let's make Juju use that too - same dep, different import syntax.